ingestion-beam/src/test/resources/testdata/heka/crashping.heka.json

```diff
@@ -10,7 +10,7 @@
         "version": "69.0",
         "xpcomAbi": "x86-msvc"
     },
-    "clientId": "a136763f-3b9a-492d-8964-bfe783b68dc6",
+    "clientId": "0e8e41d2-d46f-4818-b071-c6605257e6aa",
```

Comment:
This test was failing before the commit. I'm assuming the json is just not up to date with the heka data for the test, but correct me if I'm wrong on that, @wlach